plugins/MauticSocialBundle/Command/MauticSocialMonitoringCommand.php

```diff
@@ -47,8 +47,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return \Symfony\Component\Console\Command\Command::SUCCESS;
         }
 
+        if (!is_numeric($batchSize)) {
+            $output->writeln('batch-size is not number.');
+
+            return 1;
```

Comment:
```suggestion
            return \Symfony\Component\Console\Command\Command::FAILURE;
```

Reply:
Though i used `self::FAILURE`, which is shorter. :)